pandas/core/arrays/datetimes.py

```diff
@@ -1425,6 +1567,10 @@ def to_julian_date(self):
                  self.nanosecond / 3600.0 / 1e+9
                  ) / 24.0)
 
+    def _to_json_values(self):
```

Comment:
is this doc-stringed? is it an official method?

Reply:
Not at all official. Added a comment.

Reply:
see my comments on the json. I would rather not add this and xfail the appropriate tests for now. There is something wrong.